# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../musl/pkg/mdadm/0002-mdadm-musl-no-__off64_t.patch
# Copyright (C) 2012 - 2013 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- a/mdadm.h	2012-11-25 01:44:21.403317766 +0100
+++ b/mdadm.h	2012-11-25 01:43:12.293717364 +0100
@@ -25,6 +25,12 @@
 #define	_GNU_SOURCE
 #define _FILE_OFFSET_BITS 64
 #include	<unistd.h>
+
+/*
+ * musl doesn't know __off64_t
+ *
+ * mdadm.h:29:1: error: unknown type name '__off64_t'
+ *
 #if !defined(__dietlibc__) && !defined(__KLIBC__)
 extern __off64_t lseek64 __P ((int __fd, __off64_t __offset, int __whence));
 #else
@@ -32,6 +38,7 @@
 # define lseek64 lseek
 # endif
 #endif
+ */
 
 #include	<sys/types.h>
 #include	<sys/stat.h>
